gnd/src/main/java/com/google/android/gnd/persistence/local/LocalValueStore.java

```diff
@@ -75,15 +75,14 @@ public int getSavedMapType(int defaultType) {
   }
 
   /**
-   * This method will return whether the user have accepted the terms or not.
-   * @return true or false depending upon the user.
+   * Returns whether the currently logged in user have accepted the terms or not.
```

Comment:
Nit: have -> has
Nit: terms -> terms of service

Reply:
updated https://github.com/google/ground-android/pull/845/commits/192d034e2859551ed63ce116147537e24a01ba22